### PR TITLE
Initial scripts for GetSystemTime

### PR DESCRIPTION
**Pull request by GetmanetsIrina**
Initial scripts for GetSystemTime

Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0120-GetSystemTime.md

### CODE REVIEW

File: test_scripts/Policies/GetSystemTime/020_GetSystemTime_mobile_cer_becomes_not_valid_sdl_cer_becomes_valid_with_PTU.lua [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app starts secure RPC service
+-- 2) Mobile certificate is up to date and sdl certificate is not up to date
+-- 3) SDL requests GetSystemTime
+-- 4) According to time from GetSystemTime response mobile certificate becomes not valid and sdl certificate becomes valid
```
Comment (dboltovskyi) [inline test_scripts/Policies/GetSystemTime/020_GetSystemTime_mobile_cer_becomes_not_valid_sdl_cer_becomes_valid_with_PTU.lua]: @GetmanetsIrina According to dates of mobile certificate (17-22) and ```systemTime``` = 20 mobile certificate still valid

File: test_sets/get_system_time.txt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+./test_scripts/Policies/GetSystemTime/001_GetSystemTime_is_sent.lua
```
Comment (dboltovskyi) [inline test_sets/get_system_time.txt]: @GetmanetsIrina What about the script for the case #5 in first table on the page with checklist?

**Comment by GetmanetsIrina** [comment]
@dboltovskyi , please find update in 94c50c237831e6367ea3d3cd2dab24a3890fc32d.

**Comment by GetmanetsIrina** [comment]
@dboltovskyi , please find update in 3a8257f378c9bb79f012a1db301d8cff2fc5d223.

**Pull request merged**